src/renderers/dom/shared/ReactInputSelection.js

```diff
@@ -30,12 +33,16 @@ function isInDocument(node) {
  */
 var ReactInputSelection = {
   hasSelectionCapabilities: function(elem) {
-    var nodeName = elem && elem.nodeName && elem.nodeName.toLowerCase();
+    if (elem === window) {
+      return false;
+    }
```

Comment:
A good example of this is if you throw a breakpoint [in this function]( https://github.com/facebook/react/blob/master/src/renderers/dom/shared/eventPlugins/ChangeEventPlugin.js#L44) and click anywhere on the page other than the target input.